Intersect (Core)/Configuration/ClientConfiguration.cs

```diff
@@ -69,58 +81,98 @@ public void Validate()
         #region Options
 
         /// <summary>
-        /// Hostname of the server to connect to
+        /// Hostname of the server that the client will connect to.
         /// </summary>
         public string Host { get; set; } = DEFAULT_HOST;
 
         /// <summary>
-        /// Port of the server to connect to
+        /// Port of the server that the client will connect to.
```

Comment:
I don't think the wording changes are needed here.

Reply:
undid these changes 👍🏽